pkg/apis/core/types.go

```diff
@@ -2866,6 +2882,14 @@ type PodSecurityContext struct {
 	// If unset, the Kubelet will not modify the ownership and permissions of any volume.
 	// +optional
 	FSGroup *int64
+	// FSGroupChangePolicy defines behavior of changing ownership and permission of the volume
+	// before being exposed inside Pod. This field will only apply to
+	// volume types which support fsGroup based ownership(and permissions).
+	// It will have no effect on ephemeral volume types such as: secret, configmaps
+	// and emptydir.If not specified - default behaviour is to always recursively change permission and ownership
```

Comment:
Nit: space after `.`.

To keep in sync wih api/core/v1/types.go

Reply:
fixed